src/Illuminate/Auth/Passwords/DatabaseTokenRepository.php

```diff
@@ -53,15 +60,17 @@ class DatabaseTokenRepository implements TokenRepositoryInterface
      * @param  string  $table
      * @param  string  $hashKey
      * @param  int  $expires
+     * @param  int $timeout
      * @return void
      */
     public function __construct(ConnectionInterface $connection, HasherContract $hasher,
-                                $table, $hashKey, $expires = 60)
+                                $table, $hashKey, $expires = 60, $timeout = 60)
     {
         $this->table = $table;
         $this->hasher = $hasher;
         $this->hashKey = $hashKey;
         $this->expires = $expires * 60;
+        $this->timeout = $timeout;
```

Comment:
And use
```php
 $this->timeout = func_num_args() >= 6 ? func_get_arg(6) : 60;
```
for 6.x

Reply:
As explained above in the previous comment, this is not needed.